### PR TITLE
LPS-126288

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by WebContentWithCustomStructures#AddWCWithFirstComplexStructure

**Comment by alexcandidos** [comment]
Hi @Tim-Cao , thanks for the review! Discussing with @emanuelfisilva , we'll keep them as is in the Data Engine scope.

**Comment by Tim-Cao** [comment]
Got it

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 1]
```diff
@@ -35,12 +35,60 @@ definition {
...
+
+		Navigator.gotoNavTab(navTab = "Web Content");
+
+		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+
+		WebContent.addWithStructureCP(
+			structureName = "WC Structure Name",
+			webContentTitle = "WC Web Content Title");
+
+		Type(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:73]: Use existing macro

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 2]
```diff
@@ -35,12 +35,60 @@ definition {
...
+			locator1 = "DataEngineBuilder#WEB_CONTENT_TEXT_FIELD",
+			value1 = "Test Editing");
+
+		Type(
```
Comment (Tim-Cao) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:77]: Use existing macro

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3631
View total diff https://github.com/liferay-echo/liferay-portal/pull/3631/commits/3650d80f2820382ad4eb9d52df226efb72bdb90c

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**